src/dials/command_line/predict.py

```diff
@@ -112,9 +115,17 @@ def run(self, args=None):
             predicted_all.extend(predicted)
 
         # if we are not ignoring shadows, look for reflections in the masked
-        # region, see https://github.com/dials/dials/issues/349
-
+        # region, see https://github.com/dials/dials/issues/349. In this case
+        # we need access to the image data
         if not params.ignore_shadows:
+            try:
+                experiments = ExperimentListFactory.from_json(
```

Comment:
This makes me think we need a better way in the API of making this change

Not suggesting that we change it _here_ just that the `ExperimentList` needs to have an `add_format()` or something